charts/compliance-operator/values.yaml

```
@@ -0,0 +1,64 @@
+acm-operatorpolicy:
+  subscription:
+    channel: stable
+    name: compliance-operator-sub
+    namespace: openshift-compliance
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+    config:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      env:
+      - name: PLATFORM
+        value: "HyperShift"
+
+  operatorGroup:
+    name: compliance-operator
+    namespace: openshift-compliance
+    targetNamespaces:
+    - openshift-compliance
+
+  upgradeApproval: Manual
```

Set to automatic for managed approvals

```suggestion
  upgradeApproval: Automatic
```